upstream-sha256: 927fe8b2b2b5083306d226e02a3723731a14c75844bcb001364750bf8408eb32
actual-sha256: 9743c90f27a2d2a336280ac0b90cbd571533d4b9a4509c2abf5fa58f4efe5abd
--- upstream.expected
+++ tolk.actual
@@ -132,38 +132,70 @@
     float val61 = (*(buf0+7));
     float val62 = (*(buf0+8));
     float val63 = (*(buf0+9));
-    float alu66 = ((val32<0.0f)?0.0f:val32);
-    float alu67 = ((val33<0.0f)?0.0f:val33);
-    float alu68 = ((val44<0.0f)?0.0f:val44);
-    float alu69 = ((val55<0.0f)?0.0f:val55);
-    float alu70 = ((val58<0.0f)?0.0f:val58);
-    float alu71 = ((val59<0.0f)?0.0f:val59);
-    float alu72 = ((val60<0.0f)?0.0f:val60);
-    float alu73 = ((val61<0.0f)?0.0f:val61);
-    float alu74 = ((val62<0.0f)?0.0f:val62);
-    float alu75 = ((val63<0.0f)?0.0f:val63);
-    float alu76 = ((val34<0.0f)?0.0f:val34);
-    float alu77 = ((val35<0.0f)?0.0f:val35);
-    float alu78 = ((val36<0.0f)?0.0f:val36);
-    float alu79 = ((val37<0.0f)?0.0f:val37);
-    float alu80 = ((val38<0.0f)?0.0f:val38);
-    float alu81 = ((val39<0.0f)?0.0f:val39);
-    float alu82 = ((val40<0.0f)?0.0f:val40);
-    float alu83 = ((val41<0.0f)?0.0f:val41);
-    float alu84 = ((val42<0.0f)?0.0f:val42);
-    float alu85 = ((val43<0.0f)?0.0f:val43);
-    float alu86 = ((val45<0.0f)?0.0f:val45);
-    float alu87 = ((val46<0.0f)?0.0f:val46);
-    float alu88 = ((val47<0.0f)?0.0f:val47);
-    float alu89 = ((val48<0.0f)?0.0f:val48);
-    float alu90 = ((val49<0.0f)?0.0f:val49);
-    float alu91 = ((val50<0.0f)?0.0f:val50);
-    float alu92 = ((val51<0.0f)?0.0f:val51);
-    float alu93 = ((val52<0.0f)?0.0f:val52);
-    float alu94 = ((val53<0.0f)?0.0f:val53);
-    float alu95 = ((val54<0.0f)?0.0f:val54);
-    float alu96 = ((val56<0.0f)?0.0f:val56);
-    float alu97 = ((val57<0.0f)?0.0f:val57);
-    *(data0_32+Lidx2) = (alu66+alu67+alu68+alu69+alu70+alu71+alu72+alu73+alu74+alu75+alu76+alu77+alu78+alu79+alu80+alu81+alu82+alu83+alu84+alu85+alu86+alu87+alu88+alu89+alu90+alu91+alu92+alu93+alu94+alu95+alu96+alu97);
+    float alu66 = ((val32!=val32)?val32:0.0f);
+    float alu67 = ((0.0f<val32)?val32:alu66);
+    float alu68 = ((val33!=val33)?val33:0.0f);
+    float alu69 = ((0.0f<val33)?val33:alu68);
+    float alu70 = ((val44!=val44)?val44:0.0f);
+    float alu71 = ((0.0f<val44)?val44:alu70);
+    float alu72 = ((val55!=val55)?val55:0.0f);
+    float alu73 = ((0.0f<val55)?val55:alu72);
+    float alu74 = ((val58!=val58)?val58:0.0f);
+    float alu75 = ((0.0f<val58)?val58:alu74);
+    float alu76 = ((val59!=val59)?val59:0.0f);
+    float alu77 = ((0.0f<val59)?val59:alu76);
+    float alu78 = ((val60!=val60)?val60:0.0f);
+    float alu79 = ((0.0f<val60)?val60:alu78);
+    float alu80 = ((val61!=val61)?val61:0.0f);
+    float alu81 = ((0.0f<val61)?val61:alu80);
+    float alu82 = ((val62!=val62)?val62:0.0f);
+    float alu83 = ((0.0f<val62)?val62:alu82);
+    float alu84 = ((val63!=val63)?val63:0.0f);
+    float alu85 = ((0.0f<val63)?val63:alu84);
+    float alu86 = ((val34!=val34)?val34:0.0f);
+    float alu87 = ((0.0f<val34)?val34:alu86);
+    float alu88 = ((val35!=val35)?val35:0.0f);
+    float alu89 = ((0.0f<val35)?val35:alu88);
+    float alu90 = ((val36!=val36)?val36:0.0f);
+    float alu91 = ((0.0f<val36)?val36:alu90);
+    float alu92 = ((val37!=val37)?val37:0.0f);
+    float alu93 = ((0.0f<val37)?val37:alu92);
+    float alu94 = ((val38!=val38)?val38:0.0f);
+    float alu95 = ((0.0f<val38)?val38:alu94);
+    float alu96 = ((val39!=val39)?val39:0.0f);
+    float alu97 = ((0.0f<val39)?val39:alu96);
+    float alu98 = ((val40!=val40)?val40:0.0f);
+    float alu99 = ((0.0f<val40)?val40:alu98);
+    float alu100 = ((val41!=val41)?val41:0.0f);
+    float alu101 = ((0.0f<val41)?val41:alu100);
+    float alu102 = ((val42!=val42)?val42:0.0f);
+    float alu103 = ((0.0f<val42)?val42:alu102);
+    float alu104 = ((val43!=val43)?val43:0.0f);
+    float alu105 = ((0.0f<val43)?val43:alu104);
+    float alu106 = ((val45!=val45)?val45:0.0f);
+    float alu107 = ((0.0f<val45)?val45:alu106);
+    float alu108 = ((val46!=val46)?val46:0.0f);
+    float alu109 = ((0.0f<val46)?val46:alu108);
+    float alu110 = ((val47!=val47)?val47:0.0f);
+    float alu111 = ((0.0f<val47)?val47:alu110);
+    float alu112 = ((val48!=val48)?val48:0.0f);
+    float alu113 = ((0.0f<val48)?val48:alu112);
+    float alu114 = ((val49!=val49)?val49:0.0f);
+    float alu115 = ((0.0f<val49)?val49:alu114);
+    float alu116 = ((val50!=val50)?val50:0.0f);
+    float alu117 = ((0.0f<val50)?val50:alu116);
+    float alu118 = ((val51!=val51)?val51:0.0f);
+    float alu119 = ((0.0f<val51)?val51:alu118);
+    float alu120 = ((val52!=val52)?val52:0.0f);
+    float alu121 = ((0.0f<val52)?val52:alu120);
+    float alu122 = ((val53!=val53)?val53:0.0f);
+    float alu123 = ((0.0f<val53)?val53:alu122);
+    float alu124 = ((val54!=val54)?val54:0.0f);
+    float alu125 = ((0.0f<val54)?val54:alu124);
+    float alu126 = ((val56!=val56)?val56:0.0f);
+    float alu127 = ((0.0f<val56)?val56:alu126);
+    float alu128 = ((val57!=val57)?val57:0.0f);
+    float alu129 = ((0.0f<val57)?val57:alu128);
+    *(data0_32+Lidx2) = (0.0f+alu67+alu69+alu71+alu73+alu75+alu77+alu79+alu81+alu83+alu85+alu87+alu89+alu91+alu93+alu95+alu97+alu99+alu101+alu103+alu105+alu107+alu109+alu111+alu113+alu115+alu117+alu119+alu121+alu123+alu125+alu127+alu129);
   }
 }
